Issueneter.Filters/PredefinedFilters/AuthorFilter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Issueneter.Domain.Models;
+
+namespace Issueneter.Filters.PredefinedFilters;
+
+public class AuthorFilter : IFilter<Issue>, IFilter<PullRequest>
+{
+    public string Value { get; set; }
+
+    public AuthorFilter()
+    {
+        
+    }
```

Comment:
Может лучше в одну строчку?
```suggestion
    public AuthorFilter() { }
```

Reply:
Потом